portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uisections/Portlet.path

```diff
@@ -112,7 +112,7 @@
 </tr>
 <tr>
 	<td>PORTLET_CONTENT</td>
-	<td>//section[@id='content']//div[contains(@class,'portlet-content')]</td>
+	<td>//section[@id='content']//div[contains(@class,'portlet-content') or contains(@class,'journal-content-article')]</td>
```

Comment:
Just a double check of if this also works for the tests that not affected?

Reply:
Since no root cause found, could you try if the exist locator _PageEditor#CONTENT_PAGE_WEB_CONTENT_CONTENT_ work? That would be a more safe way to fix the issue if the locator works, thanks.